src/graph/RaytraceNode.cpp

```diff
@@ -27,6 +27,11 @@ void RaytraceNode::validate()
 
 	raysNode = getValidInput<IRaysNode>();
 
+	if (raysNode->getRanges() == nullptr || raysNode->getRangesCount() == 0) {
```

Comment:
As I think about this, I rly do not like the idea, that we have a node (which logically is optional), but in reality is required in the pipeline. What if I wanna simulate some rays (abstract, not lidar), and I do not have/need ranges? 
this requirement seems artificial.